json_journal/charts.py

```diff
@@ -22,3 +22,22 @@ def create_pie_chart(balances, axes):
         account_names.append(account_name)
         account_balances.append(abs(account_balance))
     axes.pie(account_balances, labels=account_names)
+
+def create_line_graph(cumulative_profit, axes):
+    '''
+    Create a line graph on given axes.
+
+    Parameters
+    ----------
+    Cumulative_profit : tuple
+        Contains a list for the x axis (dates) and a list for the y axis (profits).
+    Axes : int
+        Specifed axes for plotting the line graph.
+
+    Returns
+    -------
+    Line graph ready to be shown (n.b. plt.show needs to be called on this line graph)
+    '''
+    dates = cumulative_profit[0]
+    profits = cumulative_profit[1]
```

Comment:
Best to avoid magic numbers if you can. I see two options here:
1. Pass the dates and profits as separate arguments, or
2. Unpack the tuple (see [sequence unpacking](https://docs.python.org/3/tutorial/datastructures.html#tuples-and-sequences)).

Reply:
done